app/containers/BalancesContainer/index.js

```diff
@@ -0,0 +1,74 @@
+/*
+ *
+ * BalancesContainer
+ *
+ */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import Helmet from 'react-helmet';
+import selectBalancesContainer from './selectors';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+import styles from './styles.css';
+import { Token } from '../../utils/Wallet/Token'
+import W from '../../utils/Wallet/Wallet'
```

Comment:
It's not global, it's for this file specifically. Regardless, `Wallet` is the most explicit name. Don't you agree?